racket/collects/setup/unixstyle-install.rkt

```diff
@@ -259,7 +259,9 @@
                                                            r))))])
     (cond [(or (regexp-match #rx#"^\177ELF" magic)
                (regexp-match #rx#"^\316\372\355\376" magic)
```

Comment:
Worth adding a comment (further improving over the original) that these are all Mach-O variants, 32-bit and 64-bit in both little and big endian?

Reply:
Yeah, sure, will add now.

P. S. Let’s have `ppc64` here as well, it does not hurt even if it is not built, as of now; otherwise this problem is obscure enough that someone else gonna spend a few hours unnecessarily later on trying to debug it.

Reply:
@mflatt I have added a comment. Could you take a look if this is okay?